src/OpenTelemetry.Instrumentation.Dependencies/Implementation/HttpWebRequestActivitySource.net461.cs

```diff
@@ -32,21 +33,21 @@ namespace OpenTelemetry.Instrumentation.Dependencies.Implementation
     /// Created from the System.Diagnostics.DiagnosticSource.HttpHandlerDiagnosticListener class which has some bugs and feature gaps.
     /// See https://github.com/dotnet/runtime/pull/33732 for details.
```

Comment:
This comment will become no longer relevant with shift to ActivitySource, right!

Reply:
I have that in there more to give credit to the original authors and provide some history, but open to remove.

Reply:
You can say inspired from :)

Reply:
Updated!